app.js

```diff
@@ -0,0 +1,88 @@
+const storageKey = 'awesomeBooks';
+const currentFormTitle = document.querySelector('#title');
+const currentFormAuthor = document.querySelector('#author');
+const currentFormBookAddButton = document.querySelector('#book-add-button');
+const arrBooks = [];
+
+function removeBook(id) {
+  for (let i = 0; i < arrBooks.length; i += 1) {
+    if (arrBooks[i].id === id) {
+      arrBooks.splice(i, 1);
```

Comment:
- [OPTIONAL] its very nice implementing what was asked and all are working fine. I kindly suggest you use array methods that doesn't mutate the array, e.g. [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [pop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) and so on.